### DISCUSSION TITLE
Premium

### DISCUSSION THREAD

**Discussion by dewrano** [trakt/showly#417] [General]
Hi

Is premium going to be subscription based or single payment?

**Comment by michaldrabik** [comment]
@dewrano I think I'm leaning towards monthly / yearly subscription to choose from model. 

It is easier to manage/check and it gives user some flexibility compared to one time commitment.

**Comment by dewrano** [comment]
Can you list the prices?

**Reply by michaldrabik** [reply]
The lowest it can get which would be  0.99 USD / Month and 9 USD / Year.
As a user do you think this would be reasonable?

Premium would initially introduce things like:
- light theme
- ability to set custom images for shows and movies posters/fanarts
- "Quick Rate" - an option to quickly rate episodes and movies whenever you mark them as watched
- additional widgets UI options

**Reply by dewrano** [reply]
Personally, I think that's fine, and I would be pay for that. But since Showly is open source, you'll always have people that'll just build an APK without the premium checks, no?

**Reply by michaldrabik** [reply]
Correct and I'm perfectly fine with that. The main aim of this project was never to make money.

Additionally you still need to posses some kind of knowledge to do this + create API keys + you still need do it manually on every update (instead of letting PlayStore to do this for you)

In the future If the premium subscriptions are popular enough I'm hoping to use this for example to pay for JustWatch API access and other things.

**Comment by michaldrabik** [comment]
Converted to discussion.

**Comment by seldon1000** [comment]
@michaldrabik Since you're going to add premium features, will this decision affect the app's submission to fdroid? Assuming that fdroid is still a thing (https://github.com/michaldrabik/showly-2.0/issues/46).
Izzy's repo should be fine too.

**Reply by michaldrabik** [reply]
I don't think it would stop it in any way. Worst case I can prepare version without premium features (when it comes out).

I have no experience with fdroid though not sure what is and isn't possible quite yet.

**Reply by seldon1000** [reply]
Understood. I use fdroid for some foss apps but can't tell how it works behind the scenes. I just know that the official repo's rules are very strict and they don't allow any sort of proprietary code (firebase for instance). IzzyOnDroid repo should have less strict rules, but you can look into it when you're ready.
